mappings/net/minecraft/entity/mob/ShulkerEntity.mapping

```diff
@@ -3,34 +3,40 @@ CLASS net/minecraft/unmapped/C_wepsgefi net/minecraft/entity/mob/ShulkerEntity
 	FIELD f_eoeowyqr COVERED_ARMOR_BONUS_ID Ljava/util/UUID;
 	FIELD f_faxfphpy NO_COLOR B
 	FIELD f_fwpmtaxc MAX_TELEPORT_DISTANCE I
+	FIELD f_gvaaancy PEEK_PER_TICK F
 	FIELD f_hlyfzsza teleportLerpTimer I
 	FIELD f_hxfmsvrp TELEPORT_STEPS I
 	FIELD f_kdytjkmi ATTACHED_FACE Lnet/minecraft/unmapped/C_rinmcaxy;
 	FIELD f_nhqhcoaz OTHER_SHULKER_SCAN_RADIUS I
 	FIELD f_odaehnvp prevAttachedBlock Lnet/minecraft/unmapped/C_hynzadkk;
 	FIELD f_qyhdlubn PEEK_AMOUNT Lnet/minecraft/unmapped/C_rinmcaxy;
+	FIELD f_rhlsurhw FORWARD Lnet/minecraft/unmapped/C_zviwfwgf;
 	FIELD f_sxhxgvpw DEFAULT_COLOR B
 	FIELD f_tdqqxmxs openProgress F
 	FIELD f_twtbkvow COLOR Lnet/minecraft/unmapped/C_rinmcaxy;
 	FIELD f_yecffwtx COVERED_ARMOR_BONUS Lnet/minecraft/unmapped/C_hdbqsqsm;
 	FIELD f_yejiwmfa OTHER_SHULKER_LIMIT I
+	FIELD f_zdllrwfz MAX_LID_OPEN F
```

Comment:
max what?

Reply:
Hm yeah, it's Max distance the lid opens

Reply:
```suggestion
	FIELD f_zdllrwfz LID_OPEN_DISTANCE F
```

Reply:
Oh I already named it `MAX_LID_OPEN_DISTANCE`